examples/01_plotting/plot_carpet.py

```diff
@@ -83,8 +84,9 @@
     axes=ax,
     cmap="gray",
     standardize="zscore_sample",
+    title="global patterns over time separated by tissue type",
 )
 
-fig.show()
+plt.show()
```

Comment:
This is incompatible with latest mpl ?

Reply:
nope this is because if you run the example as a script the figures closes right away and you do not have time for proper visual inspection